bin/cli.js

```diff
@@ -14,8 +14,8 @@ const program = new Command();
 
 program
   .version(pkg.version)
-  .option('--logLevel <logLevel>', 'change the log level of the function', 'warn')
```

Comment:
It's quite uncommon to have camel case in option names (afaik). Is this a deliberate choice or is this a standard idiom in the node world ?

Reply:
i think that it is pretty common in the node world to have camel case in the options

Reply:
To be honest I am not sure about this. I took a look at the documentation for `commander` and that leads me to believe that `log-level` is probably the right way to go here.